### ISSUE TITLE
What version to use.

### ISSUE THREAD

**Issue by Timoulis** [nicehash/NiceHashMiner-Archived#1139]
Hello,

I wonder what version of NHM to use.
The new one 2.0.0.7 or the older 1.7.5.13 ?
I am still at the old one, and i do not know if i should go to the new one.

Regards
Tim

**Comment by ghost** [comment]
Seriously??? It clearly states, "Work in progress. Not for public use."

**Comment by Timoulis** [comment]
Yes seriously. Where did you find that. If you click the ''Check my stats online'' within  the old ver. then you go to the new NHM site where you can download ONLY the new version.
